src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js

```diff
@@ -76,6 +77,24 @@ class PresetsDetailedDialog {
         this._showCliText(false);
     }
 
+    _loadDescription() {
+        let text = this._preset.description?.join("\n");
+
+        switch(this._preset.parser) {
+            case "MARKED":
+                this._isDescriptionHtml = true;
+                text = marked.parse(text);
+                text = DOMPurify.sanitize(text);
```

Comment:
Not really sure why need dom purify here? `marked` should already be sanitized

Reply:
i was surprized too, but they say it is not sanitizing and recommending DOMPurify
https://marked.js.org/

Reply:
Ah, weird but ok 😄